LDOLayoutTemplates/Classes/LDOLayoutTemplate.h

```diff
@@ -10,11 +10,11 @@
 
 @interface LDOLayoutTemplate : NSObject
 
-@property (nonatomic, weak) IBOutlet UIView *destinationView;
-@property (nonatomic) IBOutlet UIView *templateView;
+@property (nullable, nonatomic, weak) IBOutlet UIView *destinationView;
```

Comment:
Do we want to use `nullable` or `!` here? How do we deal with missing views here?

Would it be slightly better to use `strong` in edge cases here?